### DISCUSSION TITLE
How to use the wall maintenance requester roboports

### DISCUSSION THREAD

**Discussion by sescob** [Opinionated-Blueprints/10-Books-Full-of-Rails#66] [Q&A] [answered]
Hello, I am fairly new to Factorio and using this in general. I cannot for the life of me figure out what I am missing with the inserters on the roboport for the wall maintenance. I added the wall maintenance robo port, attached the green wire to the maintenance supply requester station large pole, but I cannot create any situation for the robots to come out and I"m not sure what I am missing. I must be missing some logic here, but I am not sure where. May I have some assistance?

**Comment by sescob** [comment, marked as answer]
After some deeper trial and error, I figured out what was happening. Very nice blueprints!

**Comment by Bocian-1** [comment]
The wire isn't needed anymore on v2.19.0 - I realised there's a roboport count signal in roboports now so that is used instead
